specs/DOMContentLoaded.md

```diff
@@ -0,0 +1,119 @@
+# Background
+
+In response to consumers' requests for an event similar to the old [WebView DOMContentLoaded](https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview#mswebviewdomcontentloaded), the WebView2 team has introduced DOMContentLoaded API which indicates that the main DOM elements have finished loading.
+In this document we describe the new API. We'd appreciate your feedback.
+
+# Description
+We propose adding DOMContentLoaded to CoreWebView2. This allows the developer to have an event that fires when the DOM is loaded after the WebView2 navigates to a page.
+
+# Examples
+## Win32 C++
+```
+ScenarioDOMContentLoaded::ScenarioDOMContentLoaded(AppWindow* appWindow)
+    : m_appWindow(appWindow), m_webView(appWindow->GetWebView())
```

Comment:
AppWindow has a GetWebView?

Reply:
This is a snippet from our COM sample app which is not using WinRT APIs and which has a type named AppWindow. This is confusing. We should rename AppWindow to something else. SampleAppWindow